app/src/main/java/com/humberto/tasky/core/presentation/designsystem/components/TaskyDialog.kt

```diff
@@ -0,0 +1,52 @@
+package com.humberto.tasky.core.presentation.designsystem.components
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.RowScope
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.window.Dialog
+
+@Composable
+fun TaskyDialog(
+    onDismiss: () -> Unit,
+    dialogTitle: @Composable () -> Unit,
+    dialogContent: @Composable () -> Unit,
+    primaryButton: @Composable RowScope.() -> Unit,
+    modifier: Modifier = Modifier,
+    secondaryButton: @Composable RowScope.() -> Unit = {},
+) {
```

Comment:
Minor - for usability of this Composable I would place the `dialogContent` parameter at the end to take advantage of the trailing lambda feature.

```
@Composable
fun TaskyDialog(
  onDismiss: () -> Unit,
  dialogTitle: @Composable () -> Unit,
  primaryButton: @Composable RowScope.() -> Unit,
  modifier: Modifier = Modifier,
  secondaryButton: @Composable RowScope.() -> Unit = {},
  dialogContent: @Composable () -> Unit,
) 
```

Usage:
```
TaskyDialog(/* other params */) {
  // dialogContent lambda
}
```